Trustroots/trustroots
# Dynamic team page and move board to foundation page

### Proposed Changes

* Adds new `volunteer` role
* Adds API which returns everyone in `volunteer` role
* Lists said volunteers at `/team` page in random order: profile image, name, and link to TR profile
* Moves board members from `/team` page to `/foundation` page
* Small adjustments to links and language here and there
* Pulls images from Trustroots instead of using gravatar/stored image files; that's why in screenshots board images are gone; they'll work at the actual site then.
* I'll leave static images in the folder for now and we can clean them up a bit later until they no longer are in search engine memories and such.


### Testing Instructions

* Open `/foundation` and `/team` pages and observe it looks ok on desktop and mobile
* As an admin user, open user via `/admin` and grant them volunteer role

**Current live pages:**
- https://www.trustroots.org/team
- https://www.trustroots.org/foundation

**Here are screenshots of changes:**

### Team page before
![old-team-page](https://user-images.githubusercontent.com/87168/89327399-8ac6a700-d694-11ea-93b8-cf4e77207df5.png)

### Team page after
![new-team-page](https://user-images.githubusercontent.com/87168/89327289-5f43bc80-d694-11ea-9a40-364ca65a6439.png)

### Foundation page before
![old-foundation-page](https://user-images.githubusercontent.com/87168/89327420-91edb500-d694-11ea-8c1f-cf61c3f10f20.png)

### Foundation page after
![new-foundation-page](https://user-images.githubusercontent.com/87168/89327271-581cae80-d694-11ea-9931-9e934b71da56.png)
